app/src/app/api/daily-counters/route.ts

```diff
@@ -20,8 +20,30 @@ export async function GET() {
       dailyErrands: 0,
       dailyTrainings: 0,
     });
+
+    await drizzleDB.insert(dailyReset).values({
+      id: crypto.randomUUID(), // Generate a unique ID for the reset entry
+      resetType: "daily-pvp",
+      scheduledDate: new Date().toISOString(), // Use current timestamp for scheduled date
+      executedDate: new Date().toISOString(), // Log the execution timestamp
+      status: "completed", // Mark as completed since the reset was successful
+      lastChecked: new Date().toISOString(),
+      retryCount: 0,
+      isManualOverride: false,
+    });
     return Response.json(`OK`);
   } catch (cause) {
+    await drizzleDB.insert(dailyReset).values({
```

Comment:
I think all the inserts could probably have been abstracted into a separate function just taking a resetType.. also no need to specify values which already have a default value